apps/lfx-one/src/app/shared/components/lens-switcher/lens-switcher.component.scss

```
@@ -27,9 +27,44 @@
+  }
+
+  // Remove default content padding
+  .p-popover-content {
+    @apply p-0;
+  }
+
+  // PrimeNG sets position/left/top inline via absolutePosition(); !important is
+  // required to override. Fixed keeps the menu put while the page scrolls.
+  // stylelint-disable-next-line declaration-no-important
+  position: fixed !important;
+  // Just to the right of the rail: 56px = 48px rail + 8px gap.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)